README.md

```diff
@@ -19,7 +19,9 @@ Download native binaries of Cryptomator on [cryptomator.org](https://cryptomator
 - Client-side: No accounts, no data shared with any online service
 - Totally transparent: Just work on the virtual drive as if it were a USB flash drive
 - AES encryption with 256-bit key length
-- Filenames get encrypted, too
+- File names get encrypted
+- File metadata gets encrypted
```

Comment:
This is no longer true since [version 1.2.0](https://github.com/cryptomator/cryptomator/releases/tag/1.2.0) removed file size obfuscation. Other attributes such as modification times and owner can not be obfuscated without breaking access to ciphertext files and synchronization functionality.